docs/local_dev.md

```diff
@@ -179,3 +179,115 @@ kubectl delete cluster $CLUSTER_NAME
 docker rm -f $HOST_NAME
 kind delete cluster
 ```
+
+# Host Agent Installer
+The installer is responsible for detecting the BYOH OS, downloading a BYOH bundle and installing/uninstalling it.
+
+## Supported OS and Kubernetes
+The current list of supported tuples of OS, kubernetes Version, BYOH Bundle Name can be retrieved with:
+```shell
+./cli --list-supported
+```
+An example output looks like:
+<table>
+    <tr>
+        <td>OS</td>
+        <td>K8S Version</td>
+        <td>BYOH Bundle Name</td>
+    </tr>
+    <tr>
+        <td>Ubuntu_20.04.*_x86-64</td>
+        <td>v1.22.3</td>
+        <td>byoh-bundle-ubuntu_20.04.1_x86-64_k8s_v1.22.3</td>
+    </tr>
+</table>
+The '*' in OS means that all Ubuntu 20.04 patches will be handled by this BYOH bundle.
+
+## Pre-requisites
+As of writing this, the following packages must be pre-installed on the BYOH host:
+- socat
+- ebtables
+- ethtool
+- conntrack
+```shell
+sudo apt-get install socat ebtables ethtool conntrack
+```
+
+## Creating a BYOH Bundle
+### Kubernetes Ingredients
+Optional. This step describes downloading kubernetes host components for Debian.
+```shell
+# Build docker image
+(cd agent/installer/bundle_builder/ingredients/deb/ && docker build -t byoh-ingredients-deb .)
+
+# Create a directory for the ingredients and download to it
+(mkdir -p byoh-ingredients-download && docker run --rm -v `pwd`/byoh-ingredients-download:/ingredients byoh-ingredients-deb)
```

Comment:
Naive question: I was more aware of the usage `-v ${PWD}:/home/somepath` for volume mounts. But above is also okay if there are no downsides.

Reply:
I guess both are OK for an example.